tools/mo/unit_tests/mo/convert/meta_data_test.py

```diff
@@ -62,7 +60,6 @@ def create_onnx_model():
 
         def ref_meta_data():
             return {
-                'MO_version': get_version(),
```

Comment:
why is this changed? I propose do not change MO unit-tests unless they are move to OVC unit-tests.

Reply:
It was changed because I changed the import to ovc.convert_model which does not have MO_version, only runtime version. It looks redundant to have both, as they are in the same package.

As now we have unit tests in ovc I can remove this change.